### PR TITLE
fix(form-builder): fix default values for table questions

### PR DESCRIPTION
**Pull request by anehx**

### CODE REVIEW

**Comment by anehx** [comment]
Pipelines will succeed as soon as #1469 is merged and rebased

File: addon/components/cf-field/input/table.js [hunk 1]
```diff
@@ -21,6 +21,10 @@ export default Component.extend({
   documentToEdit: null,
   documentToDelete: null,
 
+  parseDocument(raw) {
+    return parseDocument(raw);
+  },
```
Comment (czosel) [inline addon/components/cf-field/input/table.js:26]: Why the indirection?

Comment (anehx) [reply addon/components/cf-field/input/table.js:26]: Because we need to override it in the default table component

Comment (czosel) [reply addon/components/cf-field/input/table.js:26]: Ah of course, thanks!

File: tests/integration/components/cfb-form-editor/question/default/table-test.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    setupRenderingTest(hooks);
+
+    test("it renders", async function (assert) {
+      // Set any properties with this.set('myProperty', 'value');
+      // Handle any actions with this.set('myAction', function(val) { ... });
+
+      await render(hbs`<CfbFormEditor::Question::Default::Table />`);
+
+      assert.dom(this.element).hasText("");
+    });
```
Comment (czosel) [inline tests/integration/components/cfb-form-editor/question/default/table-test.js:18]: Would it be possible to write a meaningful test for this?

**Pull request merged**